app/domain/authentication/authn_jwt/jwt_configuration_dummy_vendor.rb

```diff
@@ -2,16 +2,16 @@ module Authentication
   module AuthnJwt
     # Mock JWTConfiguration class to use it to develop other part in the jwt authenticator
     class JWTConfigurationDummyVendor < JWTConfigurationInterface
-      def get_identity
-        return "cucumber"
+      def get_conjur_id
```

Comment:
`get_` prefix is redundant in ruby. just `conjur_id` is enough.

Reply:
Done